### PR TITLE
Add `drill`, an http load testing application

### PR DESCRIPTION
**Pull request by fcsonline**
changelog: Add `drill`, an HTTP load testing application

An alternative to HTTP performance applications like jMeter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,12 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 
 * [gitui](https://github.com/extrawurst/gitui) - Blazing fast terminal-ui for git written in Rust 🦀
 
+### Performance
+
+#### jMeter
```
Comment (TaKO8Ki) [inline README.md]: Could you add GitHub repository url?

```suggestion
#### [jMeter](https://github.com/apache/jmeter)
```

File: README.md [hunk 2]
```diff
@@ -183,6 +183,12 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 
 * [gitui](https://github.com/extrawurst/gitui) - Blazing fast terminal-ui for git written in Rust 🦀
 
+### Performance
```
Comment (TaKO8Ki) [inline README.md]: Could you also add `Performance` to table of contents?

Comment (fcsonline) [reply README.md]: Sure!

Comment (fcsonline) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
   - [Text editors](#text-editors)
   - [Text processing](#text-processing)
   - [Utilities](#utilities)
+  - [Performance](#performance)
```
Comment (TaKO8Ki) [inline README.md]: Could you follow alphabet order? In this case, `Performance` should be added between `Container` and `System tools`.

Comment (fcsonline) [reply README.md]: Done

**Review by TaKO8Ki — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**